Assets/MixedRealityToolkit/_Core/Devices/OpenVR/OpenVRDeviceManager.cs

```diff
@@ -141,6 +141,9 @@ private static SupportedControllerType CurrentControllerType
                         case "Vive Knuckles - Left":                                            // TODO: Yet to test
                         case "Vive Knuckles- Right":                                            // TODO: Yet to test
                             return SupportedControllerType.ViveKnuckles;
+                        case "OpenVR Controller(WindowsMR: 0x045E/0x065B/0/1) - Left":
+                        case "OpenVR Controller(WindowsMR: 0x045E/0x065B/0/2) - Right":
```

Comment:
![image](https://user-images.githubusercontent.com/3580640/43867722-3f7e09d0-9b1f-11e8-923c-228fbd6c140f.png)

Mine comes through slightly differently (Samsung controllers, which might be why?).

Reply:
I suppose we could do a partial match here.

Reply:
I wonder what the hex values refer to.

Reply:
Good question

Reply:
>I suppose we could do a partial match here.

Possibly, but we might need to take into account future controllers with different button configurations that might match our partial match.

Reply:
I wonder what is building this string... Hopefully when we switch to the native plugin for input we'll get better info about the controllers